### DISCUSSION TITLE
Add fsnotes to the Note Taking comparison on https://www.reddit.com/r/macapps

### DISCUSSION THREAD

**Discussion by apastuszak** [glushchenko/fsnotes#1611] [General]
/r/macapps contains a list of note taking apps and has a comparison spreadsheet.

This link [https://forms.gle/cJwqGjkJ1GYgGE1o8](https://forms.gle/cJwqGjkJ1GYgGE1o8) will take you to the survey to get added to the spreadsheet.

I don't know enough about the app to complete the survey. I'm a new user. I was hoping one of the developers or a more power user might be able to complete this survey.

**Comment by gingerbeardman** [comment]
@glushchenko

**Comment by glushchenko** [comment]
I don't have time for this
